test/e2e/tests/test_table.py

```diff
@@ -33,11 +33,45 @@
 RESOURCE_PLURAL = "tables"
 
 DELETE_WAIT_AFTER_SECONDS = 15
-UPDATE_TAGS_WAIT_AFTER_SECONDS = 5
+MODIFY_WAIT_AFTER_SECONDS = 5
 
 @pytest.fixture(scope="module")
-def dynamodb_client():
-    return boto3.client("dynamodb")
+def forum_table():
```

Comment:
In the future, when making major refactoring-of-tests changes like this, it's best to do that in a separate PR. That way reviewers don't have the cognitive load of sifting through the test changes trying to identify what changes were specific to the functional additions/modifications in the PR (in this case, the addition of TTL)